haas/model.py

```diff
@@ -49,13 +48,20 @@
 def init_db(uri=None):
     """Start up the DB connection.
 
+    If `create` is True, this will generate the schema for the database, and
+    perform initial population of tables.
+
```

Comment:
Where did this come from? I think I deleted this phrasing as part of my migrations work, perhaps it snuck back in as part of a botched merge? The  `create` argument doesn't exist, so this is obviously wrong.